src/test/java/net/minecraftforge/test/BrewingRecipeRegistryTest.java

```diff
@@ -21,19 +21,19 @@ public void init(FMLInitializationEvent event)
         // The following adds a recipe that brews a piece of rotten flesh "into" a diamond sword resulting in a diamond hoe
         BrewingRecipeRegistry.addRecipe(new ItemStack(Items.diamond_sword), new ItemStack(Items.rotten_flesh), new ItemStack(Items.diamond_hoe));
 
-        ItemStack output0 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.diamond_sword), new ItemStack(Items.rotten_flesh));
+        ItemStack output0 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.diamond_sword), new ItemStack(Items.rotten_flesh), null);
         if(output0.getItem() == Items.diamond_hoe)
             System.out.println("Recipe succefully registered and working. Diamond Hoe obtained.");
         
         // Testing if OreDictionary support is working. Register a recipe that brews a gemDiamond into a gold sword resulting in a diamond sword
         BrewingRecipeRegistry.addRecipe(new ItemStack(Items.golden_sword), "gemDiamond", new ItemStack(Items.diamond_sword));
         
-        ItemStack output1 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.golden_sword), new ItemStack(Items.diamond));
+        ItemStack output1 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.golden_sword), new ItemStack(Items.diamond), null);
         if(output1.getItem() == Items.diamond_sword)
             System.out.println("Recipe succefully registered and working. Diamond Sword obtained.");
         
         // In vanilla, brewing netherwart into a water bottle results in an awkward potion (with metadata 16). The following tests if that still happens
-        ItemStack output2 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.potionitem, 1, 0), new ItemStack(Items.nether_wart));
+        ItemStack output2 = BrewingRecipeRegistry.getOutput(new ItemStack(Items.potionitem, 1, 0), new ItemStack(Items.nether_wart), null);
```

Comment:
If you call with `null` as a `TileEntity` parameter, should you add `@Nullable` on the `TileEntity` parameter in the `getOutput` method in the interface to notify users that the tile entity might be null?